docs/DeveloperGuide.md

```diff
@@ -290,6 +337,8 @@ Here is the sequence diagram to better illustrate the lifecycle of a browse comm
 
 ![Browse Sequence Diagram](images/Browse-SequenceDiagram.png)
 
+<br/>
```

Comment:
Thanks for helping me fix the formatting for my images!

Reply:
Nope :)